app/controllers/users_controller.rb

```diff
@@ -10,6 +10,7 @@ def new
 
   def create
     @user = User.new(params[:user])
+    @user.bio ||= ''
```

Comment:
During Testing of **'user creation'**  : https://github.com/publiclab/plots2/blob/348334366f6ccf499a80498e5e06667417f8d764/test/unit/user_test.rb#L4 
we are creating user over there , so i guess this **@user.bio ||= ' '**  statement here will not run (for testing).
So i guess we should either give **bio: 'something'** attribute here :
https://github.com/publiclab/plots2/blob/348334366f6ccf499a80498e5e06667417f8d764/test/unit/user_test.rb#L5
OR
maybe here (because this function is called before the user is created) : 
https://github.com/publiclab/plots2/blob/348334366f6ccf499a80498e5e06667417f8d764/app/models/user.rb#L47

What do you think sir ? :smile:
Thank you !

Reply:
Oh thank you for your help -- i've been traveling and working on events recently and have been slow on responding to pull requests, but will do my best to catch up!

Reply:
`user.rb#47` was causing me trouble in tests, as I've rearranged the ordering a bit here. But I think you're right we have to do it in the model. Let me try again here, thanks for the assistance!